wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -89,12 +99,12 @@ const SwipingContainer = (props) => {
           <StyledDiv padding={2}>
             <MovieDetail
               title={title}
-              year="2016"
-              lengthOfMovie="2h 28m"
-              rating="8/10"
-              genre="Thriller"
+              year={year}
+              lengthOfMovie={lengthOfMovie}
+              rating={rating}
+              genre={genre}
```

Comment:
What are your thoughts on having a label “Media Type: “ with the type beside it? So the field is clearer to the user. Let me know what you think!

Reply:
Agreed, resolved with the previous comment. Thanks!